src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final Random random = new Random();
+        final ColorSupplier colorSupplier = new ColorSupplier(random);
```

Comment:
```suggestion
        Random random = new Random();
        ColorSupplier colorSupplier = new ColorSupplier(random);
```
there is no real need to make them final